source/rafcon/gui/design_config.yaml

```diff
@@ -0,0 +1,16 @@
+TYPE: DESIGN_CONFIG
+USE_CUSTOM_DESIGN: False
```

Comment:
Is this option still required?

Reply:
yes, I see it is still used in 3 other places in gui/config.py